client/src/api/UserAPI.ts

```diff
@@ -23,3 +30,25 @@ export const signup = async (data: SignupVals): Promise<AxiosResponse> => {
     throw error;
   }
 };
+
+/**
+ * API login request
+ *
+ * @param {string} username - Username
+ * @param {string} password - Password
+ * @returns {AxiosResponse}
+ * @throws {AxiosError}
+ */
+export const loginRequest = async (
+  data: LoginVals,
+): Promise<AxiosResponse<LoginResponse>> => {
+  try {
+    const response = await axios.post<LoginResponse>(
+      'http://localhost:7005/users/login',
```

Comment:
Would this adress work if we run it in docker(?) I think we should be getting the main path `https://localhost:7005` from a function that checks where it is

Reply:
yes, this is the exact address that you send your requests to with docker when the server is running